app/models/work.rb

```diff
@@ -0,0 +1,22 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+
+  validates :title, presence: true
+  validates :category, presence: true, inclusion: {in: %w(album movie book)}
+
+  def self.top_media
+    if self.count == 0
+      top_media = nil
+    else
+      works = self.all.sort_by { |work| work.votes.length }.reverse!
+      top_media = works.first
+    end
+    return top_media
```

Comment:
_Do_ we need to check if `self.count` is zero?

Consider this: calling `.first` on an empty array returns `nil` (aka in IRB, we can check: `[].first` returns `nil`)

So consider this refactor, which passes all of the tests still:

```ruby
  def self.top_media
    works = self.all.sort_by { |work| work.votes.length }.reverse!
    top_media = works.first
    return top_media
  end
```